### DISCUSSION TITLE
Incorrect display of UTF-8 symbols

### DISCUSSION THREAD

**Discussion by Alewep** [mozman/ezdxf#1211] [Issues] [answered]
I created a minimal DXF file to reproduce the bug.  
[[test_0.zip](https://github.com/user-attachments/files/17933130/test_0.zip)](https://github.com/user-attachments/files/17933130/test_0.zip)

When I display the drawing in AutoCAD, the symbol ∅ is displayed correctly.  
However, when using the command `ezdxf view test_0.dxf`, the symbol is displayed as a square.

**Using ezdxf:**  
![image](https://github.com/user-attachments/assets/e638f161-cb60-437a-a766-d12adc533a37)

**Using AutoCAD:**  
![image](https://github.com/user-attachments/assets/2f146aad-be5d-4cf8-99d8-e140b84911f5)

**Comment by mozman** [comment]
Font problem, read this: https://ezdxf.mozman.at/docs/tools/fonts.html

**Comment by mozman** [comment, marked as answer]
If a font doesn't include a glyph, AutoCAD uses a glyph from the default unicode font, `ezdxf` doesn't do this. 

Solution: use a font which supports all the glyphs you need.

**Reply by Alewep** [reply]
Thank you! I solved my problem. I needed to replace `arial.ttf` with `ARIALUNI.ttf`
